frontend/src/leaderboard.js

```diff
@@ -1,36 +1,87 @@
-const ACCOUNTS_URL = 'http://localhost:3000/accounts';
 
-document.getElementById('leaderboard-button').addEventListener('click', () => {
-  const leaderboard = document.getElementById('leaderboard-window');
-  while (leaderboard.firstChild) {
-    leaderboard.removeChild(leaderboard.firstChild);
+class Leaderboard {
+  constructor() {
+    this.leaderboardDiv = document.getElementById('users-container');
+    this.leaderboardWindow = document.getElementById('leaderboard-window');
+    this.leaderboardToggleButton = document.getElementById('leaderboard-toggle-button');
```

Comment:
I am not sure if we want to have these tightly coupled. Basically my issue with this is that the file leaderboard.js should be as self contained as possible, not having intimate knowledge of our other components. The way these are set up will only work now for exactly how we have our index.html set up.

Possible resolutions would be passing in the names of these three, or setting them within the functions they are used. (Or doing nothing for now but trying to think of a solution)

Reply:
good point, hmm... ill slack u